srcpkgs/alsa-ucm-conf/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'alsa-ucm-conf'
+pkgname=alsa-ucm-conf
+version=1.2.4
+revision=1
+archs="i686* x86_64*"
+depends="alsa-lib"
+short_desc="ALSA Use Case Manager topology configurations"
+maintainer="cinerea0 <cinerea0@protonmail.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/alsa-project/alsa-ucm-conf"
+distfiles="https://www.alsa-project.org/files/pub/lib/${pkgname}-${version}.tar.bz2"
+checksum=2c3b535c77dcb9aaf62a61f4f8324f1ab184162f105f7ec9ed1e37c742fcd340
+
+do_install() {
+	vinstall ucm2/README.md 0755 /usr/share/doc/${pkgname}
+	rm ucm2/README.md
+	vmkdir /usr/share/alsa/ucm2 0755
+	for item in ucm2/*; do
+		vcopy ${item} /usr/share/alsa/ucm2
+	done
```

Comment:
Could this loop simply `vinstall` the folder into `usr/share/alsa` ? That would avoid the need for `vmkdir`.

Reply:
Would that overwrite other files in the `ucm2` folder during installation?

Reply:
No. The result should be exactly the same.

Reply:
I'm getting the following error after replacing the lines selected above with `vinstall ucm2 0755 usr/share/alsa`:
```
/usr/bin/install: omitting directory 'ucm2'
=> ERROR: alsa-ucm-conf-1.2.4_1: do_install: 'install -Dm${mode} "${file}" "${_destdir}/${targetdir}/${file##*/}"' exited with 1
=> ERROR:   in _vinstall() at common/environment/setup/install.sh:171
=> ERROR:   in _noglob_helper() at common/environment/setup/install.sh:12
=> ERROR:   in do_install() at srcpkgs/alsa-ucm-conf/template:19
```

Reply:
I might be mistaken and the correct thing to do would be to use `vcopy` for the entire folder, then. (which avoid the loop)

Reply:
That works! Is it okay to leave off the installation mode, because `vcopy` doesn't allow setting that.

Reply:
Should be okay, assuming the permissions in the tarball itself were reasonable.

Reply:
I'm having a problem. Removing the `vmkdir` line and copying directly into the `alsa` directory just spews the contents of `ucm2` all over the `alsa` directory. Making the `ucm2` directory first and then copying the folder into it results in the files being located in `/usr/share/alsa/ucm2/ucm2`. Shouldn't there be a middle ground here?

Reply:
I believe you need to `vmkdir usr/share/alsa` :)

Reply:
Yep, that did it. Pushing an update shortly.